extensions/interactions/AlgebraicExpressionInput/protractor.js

```diff
@@ -19,8 +19,17 @@
 
 var objects = require(process.cwd() + '/extensions/objects/protractor.js');
 
-var customizeInteraction = function() {
-  // There are no customizations.
+var customizeInteraction = async function(elem, customLetters) {
+  expect(await elem.element(
+    by.css('.protractor-test-custom-letters-div')).isPresent()).toBe(true);
+  for (let letter of customLetters) {
+    await elem.element(by.buttonText('abc')).click();
```

Comment:
Please make sure the element has loaded before trying to click it. You can use `protractor_utils/action.js` to do this easily

Reply:
This is a common problem we see in e2e test PR reviews. You can use [this checklist](https://github.com/oppia/oppia/wiki/End-to-End-Tests#codeowner-checks) to check for these common problems when you're writing PRs

Reply:
I understand, I've fixed this now and I will keep this in mind for future, thanks!